packages/dds/merge-tree/src/client.ts

```diff
@@ -975,7 +1013,8 @@ export class Client extends TypedEventEmitter<IClientEvents> {
 				this.applyAnnotateRangeOp(opArgs);
 				break;
 			}
-			case MergeTreeDeltaType.OBLITERATE: {
+			case MergeTreeDeltaType.OBLITERATE:
+			case MergeTreeDeltaType.OBLITERATE_SIDED: {
```

Comment:
i feel like there are more cases where we differentiate between op types that also need to be handled